src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+
+  static die(animal) {
+    if (animal.health === 0) {
+      const index = Animal.alive.indexOf(animal);
+
+      Animal.alive.splice(index, 1);
```

Comment:
This solution is more obvious and will allow you to filter out all the animals that already have a health level of zero or below zero. Rewrite this solution with `filter` method.